### DISCUSSION TITLE
Abort endpoint call

### DISCUSSION THREAD

**Discussion by AmoonPod** [serverpod/serverpod#443] [Q&A] [answered]
Is there a way to abort an endpoint call after it has been called from the client? 

If you do a lot of consecutive calls, you are not able to talk to the server before all the calls are completed.

**Comment by yahu1031** [comment]
@AmoonPod As of now, you can only approach this requirement using streams AFAIK which is better not recommended. If this serverpod framework uses [dio](https://pub.dev/packages/dio#cancellation) package, there may be a chance of implementing this.

**Comment by vlidholt** [comment, marked as answer]
I suppose you technically can close the client, then open a new one. That should abort all current calls.

**Reply by AndryHTC** [reply]
How to handle cases where we need to abort the call to the same endpoint/method, but not other endpoints/methods?
